docs/runtime/types/event-enum.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Event Enum"
 ---
-Substrate runtimes provide events to communicate information for end-users and clients.
+Substrate runtimes provide events to communicate information for end users and clients.
```

Comment:
Change based on [this result](https://books.google.com/ngrams/graph?content=end+user%2Cend-user&case_insensitive=on&year_start=1800&year_end=2008&corpus=15&smoothing=3&share=&direct_url=t4%3B%2Cend%20user%3B%2Cc0%3B%2Cs0%3B%3Bend%20user%3B%2Cc0%3B%3BEnd%20User%3B%2Cc0%3B%3BEnd%20user%3B%2Cc0%3B%3BEND%20USER%3B%2Cc0%3B.t4%3B%2Cend%20-%20user%3B%2Cc0%3B%2Cs0%3B%3Bend%20-%20user%3B%2Cc0%3B%3BEnd%20-%20User%3B%2Cc0%3B%3BEnd%20-%20user%3B%2Cc0%3B%3BEND%20-%20USER%3B%2Cc0#t4%3B%2Cend%20user%3B%2Cc0%3B%2Cs0%3B%3Bend%20user%3B%2Cc0%3B%3BEnd%20User%3B%2Cc0%3B%3BEnd%20user%3B%2Cc0%3B%3BEND%20USER%3B%2Cc0%3B.t4%3B%2Cend%20-%20user%3B%2Cc0%3B%2Cs0%3B%3Bend%20-%20user%3B%2Cc0%3B%3BEnd%20-%20User%3B%2Cc0%3B%3BEnd%20-%20user%3B%2Cc0%3B%3BEND%20-%20USER%3B%2Cc0).